### PR TITLE
AuthorizedKeysCommand should not query db directly

### PR DESCRIPTION
**Pull request by zeripath**
As per @lafriks comment.

AuthorizedKeysCommand should use API.

### CODE REVIEW

**Review by sapk — APPROVED** [review]

File: routers/private/internal.go [hunk 1]
```diff
@@ -76,6 +76,7 @@ func CheckUnitUser(ctx *macaron.Context) {
 // These APIs will be invoked by internal commands for example `gitea serv` and etc.
 func RegisterRoutes(m *macaron.Macaron) {
 	m.Group("/", func() {
+		m.Post("ssh/authorized_keys", AuthorizedPublicKeyByContent)
```
Comment (lunny) [inline routers/private/internal.go]: `/ssh`?

File: routers/private/key.go [hunk 2]
```diff
@@ -8,6 +8,7 @@ package private
 import (
 	"code.gitea.io/gitea/models"
 	"code.gitea.io/gitea/modules/timeutil"
+	"net/http"
```
Comment (lunny) [inline routers/private/key.go]: import sequence.

**Review by lunny — CHANGES_REQUESTED** [review]

File: routers/private/internal.go (hunk 1 above)
Comment (zeripath) [reply routers/private/internal.go]:
```suggestion
		m.Post("/ssh/authorized_keys", AuthorizedPublicKeyByContent)
```

File: routers/private/key.go (hunk 2 above)
Comment (zeripath) [reply routers/private/key.go]: Done

File: routers/private/internal.go (hunk 1 above)
Comment (zeripath) [reply routers/private/internal.go]: Done

**Review by axifive — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Pull request merged**